modules/ocf_admin/manifests/init.pp

```diff
@@ -35,4 +35,27 @@
       group  => ocfstaff,
       mode   => '0640';
   }
+
+  # Firewall Rules #
```

Comment:
It looks like this header comment is being used somewhat inconsistently. It seems unnecessary to me, but, as long as it's there, I would like it to show up in every `init.pp` with firewall rules and for it to have the same appearance everywhere (e.g. the trailing `#`, the before-and-after spacing).

Reply:
This was my doing. I'm going to get rid of them. We already leave a comment before every rule which should be enough to indicate that what follows are firewall rules.